step.yml

```diff
@@ -13,11 +13,6 @@ source_code_url: https://github.com/bitrise-io/steps-fastlane
 support_url: https://github.com/bitrise-io/steps-fastlane/issues
 host_os_tags:
   - "osx-10.10"
```

Comment:
Could you remove the host_ost_tags as well?

1. It's not used.
2. It's not in sync with the `deps`. (`host_ost_tags` says only OSX while the `deps` says OSX and Ubuntu too)